### PR TITLE
rename CustomeAccountId->CustomAccountId

### PR DESCRIPTION
**Pull request by kolov**
# Description

Rename parameter CustomeAccountId -> CustomAccountId

# How Has This Been Tested?
The AWS Pull integration is not released yet, preparing for release

# Checklist:
- [ ] I have updated the relevant component changelog(s) (The AWS Pull integration is not released yet, preparing for release)
- [ ] This change does not affect any particular component (e.g. it's readme or docs change) (The AWS Pull integration is not released yet, preparing for release)

### CODE REVIEW

File: coralogix-metrics-integration-policy/template.yaml [hunk 1]
```diff
@@ -18,7 +18,7 @@ Parameters:
   RoleName:
     Type: String
     Description: The name of the role that will be created.
-  CustomeAccountId:
+  CustomAccountId:
```
[the remark this answers is not in the document]
Comment (zhrebicek) [reply coralogix-metrics-integration-policy/template.yaml:21]: Yeah I think so @coralogix/team-cs-devops should we change that?

[the remark this answers is not in the document]
Comment (zhrebicek) [reply coralogix-metrics-integration-policy/template.yaml:21]: Maybe CoralogixEnvironmentToAWSAccountId?

[the remark this answers is not in the document]
Comment (guyrenny) [reply coralogix-metrics-integration-policy/template.yaml:21]: I think that you can change it to CoralogixEnvironment maybe
Also just make sure that if you change `CustomeAccountId` to `CustomAccountId` to update the name in the rest of the template, the Conditions section and in line 64 from what i saw are still with the old name

**Comment by zhrebicek** [comment]
Did the changes in https://github.com/coralogix/cloudformation-coralogix-aws/pull/146/files

**Pull request closed without merging**